File: Python Pattern Programs/Numeric Patterns/Pattern 34.py
size = 4
for x in range(size, -(size + 1), -1):
    for y in range(1, abs(x) + 1):
        print(" ", end="")
    p = 1
    for z in range(size, abs(x) - 1, -1):
        print(str(p) + " ", end="")
        p += 1
    print()

File: C# Pattern Programs/Numeric Patterns/Pattern 34.cs
using System;

   class Pattern_78
 {
    public static void Main(string[] args)
    {
        int size = 4;
        int p = 1; // print value

        for (int i = size; i >= -size; i--)
        {
            for (int j = 1; j <= Math.Abs(i); j++)
            {
                Console.Write(" ");
            }

            p = 1;
            for (int k = size; k >= Math.Abs(i); k--)
            {
                Console.Write(p + " ");
                p++;
            }

            Console.WriteLine();
        }
        Console.ReadKey(true);
    }
   
}
